T/TempestRemap/build_tarballs.jl

```diff
@@ -89,7 +89,8 @@ products = [
 
 dependencies = [
     Dependency("OpenBLAS32_jll"),
-    Dependency("NetCDF_jll", compat="400.902.5 - 400.999"),
+    Dependency("HDF5_jll", compat="~1.14"),
+    Dependency("NetCDF_jll", compat="400.902.5"),
```

Comment:
```suggestion
    Dependency("NetCDF_jll", compat="400.902.207"),
```

Reply:
Given https://github.com/JuliaRegistries/General/pull/84258, I'd not change NetCDF and only add hdf5 with `~1.12`.  Next version of this package we can bump hdf5